src/components/HeadFooter/HeadFooter.js

```diff
@@ -37,19 +37,29 @@ const overwrittenTheme = createTheme({
 	}
 });
 
-const useStyles = makeStyles(() => ({
-	wrapper: {
-		minHeight: '100%',
-		display: 'flex',
-		flexDirection: 'column'
-	},
-	headFooter: {
-		flexShrink: 0
-	},
-	main: {
-		flexGrow: 1
-	}
-}));
+const mobileAdaptiveThreshold = theme => theme.breakpoints.values.sm * 1.05;
```

Comment:
Out of curiosity, where did you get 1.05?

Reply:
this was copied from the other threshold i think it was in events